qemu/tests/cfg/nvdimm.cfg

```diff
@@ -100,12 +100,16 @@
         - nvdimm_negative:
             type = nvdimm_negative
             start_vm = no
+            vm_mem_limit = 8G
             size_mem = 8G
             pre_command = "dd if=/dev/zero of=${nv_backend} bs=1M count=8192"
             variants:
                 - align_max:
                     align_mem = 4G
-                    error_msg = "the alignment \(0x100000000\) (is not supported|exceeds the expected maximum alignment)"
+                    required_qemu_version = [6.0.0, )
+                    align_max_test = yes
+                    error_after_6_0 = "the alignment (0x100000000) exceeds the expected maximum alignment, memory will get fragmented and not all 'maxmem' might be usable for memory devices."
+                    error_before_6_0 = "the alignment (0x100000000) is not supported"
```

Comment:
Well, if it is this case,why do not use regex match ?
The regex match was replaced with just str in with patch https://github.com/autotest/tp-qemu/pull/3610/files 
This change need be revert i think.  Thanks!

Reply:
@yanan-fu 
If change back to regex match the case fails:
```bash
[stdlog] 2023-04-03 06:14:24,138 avocado.test INFO | Context: Check the expected error message: \'unarmed\' property must be \'on\' since memdev mem-mem1 is read-only
[stdlog] 2023-04-03 06:14:24,145 avocado.test ERROR| 
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR| Reproduced traceback from: /usr/local/lib/python3.9/site-packages/avocado_framework_plugin_vt-100.1-py3.9.egg/avocado_vt/test.py:274
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR| Traceback (most recent call last):
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|   File "/usr/local/lib/python3.9/site-packages/avocado_framework_plugin_vt-100.1-py3.9.egg/virttest/error_context.py", line 135, in new_fn
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|     return fn(*args, **kwargs)
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|   File "/root/avocado/data/avocado-vt/virttest/test-providers.d/downloads/io-github-autotest-qemu/qemu/tests/nvdimm_negative.py", line 34, in run
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|     test.fail("Can not get expected error message: %s" % error_msg)
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|   File "/usr/local/lib/python3.9/site-packages/avocado_framework-101.0-py3.9.egg/avocado/core/test.py", line 779, in wrapper
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|     return func(actual_message)
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|   File "/usr/local/lib/python3.9/site-packages/avocado_framework-101.0-py3.9.egg/avocado/core/test.py", line 795, in fail
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR|     raise exceptions.TestFail(msg)
[stdlog] 2023-04-03 06:14:24,146 avocado.test ERROR| avocado.core.exceptions.TestFail: Can not get expected error message: \'unarmed\' property must be \'on\' since memdev mem-mem1 is read-only
```